exp/go-templates/go.mod

```diff
@@ -1,5 +1,16 @@
 module github.com/kujenga/website/exp/go-templates
 
-go 1.17
+go 1.16
 
-require gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+require (
+	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible
```

Comment:
See comment here about the effect on the size of the binary: https://github.com/kujenga/website/pull/63#pullrequestreview-951282773

> After compression (as observed in FF dev tools), that translates to:
> - 2.28MB on this branch
> - 1.62MB on `main`
> 
> While unfortunate, additionally functionality seems worth the increase.